src/renderer/helpers/utils.js

```diff
@@ -247,6 +253,34 @@ export function openExternalLink(url) {
   }
 }
 
+/**
+ * Opens an internal path in the same or a new window.
+ * Optionally with query params and setting the contents of the search bar in the new window.
+ * @param {string} path the internal path to open
+ * @param {boolean} createNewWindow set to true to open a new window
+ * @param {object} query the query params to use (optional)
+ * @param {string} searchQueryText the text to show in the search bar in the new window (optional)
+ */
+export function openInternalPath(path, createNewWindow, query = {}, searchQueryText = null) {
```

Comment:
Why are we going back to positional arguments...

Reply:
As this function has 4 parameters with two of them being optional, I'm happy switching this function back to taking an object as an argument.

Reply:
Yes please
For API documentation (I assume it's JSDoc?) for option object you can take a look at https://stackoverflow.com/a/14820610/838346